components/02-elements/rating/RatingRate.vue

```diff
@@ -0,0 +1,257 @@
+    <legend id="rating-field-label">
+      {{ legend }}
+    </legend>
+    <div
+      class="rating rating--rate"
+      role="listbox"
+      aria-required="true"
+      aria-labelledby="rating-field-label"
+    >
+      <rating-rate-item
+        v-for="(item, idx) in ratingItems"
+        :id="item.id"
+        :key="item.id"
+        :aria-label="item.labelText"
+        :name="item.name"
+        :active="rating >= idx + 1"
+        :selected="rating === idx + 1"
+        @select="onSelect(idx + 1)"
+      />
+    </div>
+  </fieldset>
+</template>
+
+<script>
+  import RatingRateItem from './RatingRateItem'
+
+  export default {
+    components: {
+      RatingRateItem
+    },
+    model: {
+      prop: 'rating',
+      event: 'change'
+    },
+    props: {
+      rating: {
+        type: Number,
+        default: 0
+      },
+      ratingItems: {
+        type: Array,
+        required: true
+      },
+      legend: {
+        type: String,
+        required: true
+      }
+    },
+    methods: {
+      onSelect(selectedIndex) {
+        this.$emit('change', selectedIndex);
+      }
+    }
+  }
+</script>
+
+<style lang="scss">
+  $rating__size                   : 95px !default;
+  $rating__item-size              : $rating__size / 5 !default;
+  $rating__size--rate             : 250px !default;
+  $rating__margin--rate           : $spacer--medium !default;
+  $rating__item-size--rate        : $rating__size--rate / 5 !default;
+  $icon__active-color             : '#fab216' !default;
+  $icon__active-color--with-border: '#fab216' !default;
+  $icon__inactive-color           : '#c2c1df' !default;
+  $rating__error-margin-bottom    : $spacer !default;
+  $rating__error-color            : $red !default;
+
+  .rating {
+    width: $rating__size;
+
+
+    &:hover,
+    &:focus {
+      .rating__rate-item span:before {
+        display: block;
+      }
+    }
+
+    &__rate-item {
+      position: relative;
+      flex: 1 0 $rating__item-size;
+      overflow: hidden;
+
+      &:hover,
+      &:focus {
+        ~ .rating__rate-item span:before {
+          display: none;
+        }
+      }
+    }
+    &__star {
+      position: relative;
+      height: $rating__item-size;
+      width: $rating__size;
+
+      &:before { // inactive
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        top: 0;
+        right: 0;
+        display: block;
+        width: $rating__size;
+        content: "";
+        height: $rating__item-size;
+        background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 25 24' xmlns='http://www.w3.org/2000/svg' fill-rule='evenodd' clip-rule='evenodd' stroke-linejoin='round' stroke-miterlimit='1.414'%3E%3Cpath fill='none' d='M-2-2h24v24H-2z'/%3E%3Cpath d='M20 7.24l-7.19-.62L10 0 7.19 6.63 0 7.24l5.46 4.73L3.82 19 10 15.27 16.18 19l-1.63-7.03L20 7.24zM10 13.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L10 4.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L10 13.4z' fill='%23c2c1df'/%3E%3C/svg%3E");
```

Comment:
Let's add a ticket to refactor this, it should be handled no SVG level, without inlining it in CSS.